_posts/release/2018-07-18-accumulo-1.9.3.md

```diff
@@ -36,15 +98,152 @@ I0123abc.rf, you are probably at low risk from this bug.
 
 This issue was fixed in pull request [#562]
 
-### ~Another notable item here~ (affects ~versions~)
+### Updated Master Metrics to include FATE metrics.
+
+Added master metrics to provide a snapshot of current FATE operations.  The metrics added:
++ the number of current FATE transactions in progress, 
++ the count of child operations that have occurred on the zookeeper FATE node
++ a count of zookeeper connection errors when the snapshot is taken.  
+
+The number of child operations provides a light-weight surrogate for FATE transaction 
+progression between snapshots. The metrics are controlled with the following properties:
+
+* master.fate.metrics.enabled - default to _false_ preserve current metric reporting
+* master.fate.metrics.min.update.interval - default to _60s_ - there is a hard limit of 10s.
+
+When enabled, the metrics are published to JMX and can optionally be configured using standard 
+hadoop metrics2 configuration files.
+
+### Fixed issues with Native Maps on Fedora
+
+On Fedora 28 and 29, with latest OpenJDK 8, the C++ compiler changed and generates code that
+causes issues.  Fixed [#767] correct allocation issue in native maps [#769] and ...
 
-More description here. This was fixed in [#NUM].
+### Stop locking during compaction
 
-## Other Changes
+Compactions were acquiring the tablet lock between each key value. 
+This created unnecessary contention with other operations like scan and 
+bulk imports.  For {% ghi 1031 %} the the synchronization was removed
+by {% ghi 1032 %}. 
```

Comment:
I'm a little confused by the use of `{% ghi 1032 %}` vs. the `[#1032]` pattern that other descriptions use. Is there a difference between these two?

Reply:
The {% ghi.1031 %} are being converted to the [#xxxx] format. The different form can from different reviewers.

Reply:
Ah. I think that's some Jekyll/Liquid magic that @mikewalch set up for our site to make linking in the documentation easier. @milleruntime didn't use that when he generated the initial template for the release notes. Either way works for me. I think the Jekyll/Liquid stuff might be a bit less text to commit, though it's easier for a human to parse the Markdown format as a footnote.

Reply:
The liquid stuff is much faster write and less error prone.

Reply:
The custom liquid tags are documented here :

https://github.com/apache/accumulo-website#custom-liquid-tags

Reply:
I think there's probably some learning curve involved before "less error prone" becomes true, since it's a custom thing, instead of standard Markdown.